portal/models/user.py

```diff
@@ -1275,7 +1281,8 @@ def update_identifiers(fhir):
                 try:
                     new_id = Identifier.from_fhir(identifier)
                     if new_id in seen:
-                        abort(400, 'Duplicate identifiers found, should be unique set')
+                        abort(400, 'Duplicate identifiers found, should be '
```

Comment:
Break the line at the comma

Reply:
@ivan-c linebreak at the comma still requires the string literal to be split in two - still do it?